packages/edit-site/src/components/post-edit/index.js

```diff
@@ -78,6 +67,43 @@ function PostEditForm( { postType, postId } ) {
 		'comment_status',
 	];
 
+	const form = useMemo(
+		() => ( {
+			type: 'panel',
+			fields: [
+				'featured_media',
+				'title',
+				'status_and_visibility',
+				'author',
+				'date',
+				'slug',
+				'comment_status',
+			].filter(
+				( field ) =>
+					ids.length === 1 ||
+					fieldsWithBulkEditSupport.includes( field )
+			),
+			combinedFields: [
```

Comment:
The API for controlling [visibility](https://github.com/WordPress/gutenberg/pull/65826/) landed, but it didn't have support for combinedFields. We also have [this other PR](https://github.com/WordPress/gutenberg/pull/66531) that aims to replace combinedFields API.

How would you like to move forward? Do we merge 66531 first and come back to this one later? Implement visibility for combinedFields here? Any other approach?